### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by FBartos**
fixes: jasp-stats/jasp-test-release#1723
fixes: jasp-stats/jasp-test-release#1709 
fixes: jasp-stats/jasp-test-release#1744

addresses parts of: jasp-stats/jasp-test-release#1698

### CODE REVIEW

**Comment by FBartos** [comment]
Probably makes sense to merge to review and merge this in before: https://github.com/jasp-stats/jaspCochrane/pull/2
(I will need to update the unit tests there because some of the names and labels changed with a new scraping algorithm)

**Comment by Kucharssim** [comment]
> Probably makes sense to merge to review and merge this in before: #2
(I will need to update the unit tests there because some of the names and labels changed with a new scraping algorithm)

Just to make sure that I understand correctly: You want to review this PR before https://github.com/jasp-stats/jaspCochrane/pull/2, is that right?

**Comment by FBartos** [comment]
yes please

File: R/cochranecommon.R [hunk 1]
```diff
@@ -508,6 +508,9 @@ CochraneCommon   <- function(jaspResults, dataset, options, type) {
     else
       selectedTitles <- sapply(indexing, function(inx) inx[["titleMetaAnalysis"]])[textSearchPositive & !textSearchNegative]
 
+    if (length(selectedTitles) == 0)
```
Comment (Kucharssim) [inline R/cochranecommon.R:511]: Perhaps give user a feedback? Not crucial, but would be nice

File: inst/qml/qml_components/CochraneInput.qml [hunk 2]
```diff
@@ -156,8 +92,54 @@ Section
...
+			{label: qsTr("Log(odds ratios)"),			value: "OR"},
+			{label: qsTr("Log(Peto's odds ratios)"),	value: "POR"},
+			{label: qsTr("Log(risk ratios)"),			value: "RR"},
+			{label: qsTr("Risk differences"),			value: "RD"	}
+		]
+	}
+
 	ComponentsList
 	{
+		title:				qsTr("Selected Reviews/Meta-Analyses")
```
Comment (Kucharssim) [inline inst/qml/qml_components/CochraneInput.qml:142]: Can you make this title and the title to `selectionType` ("Select systematic reviews based on") consistent? I assume it's either both

"Selecting systematic review/meta-analyses based on" + "Selected systematic reviews/meta-analyses"
or
"Selecting Reviews/Meta-Analyses based on" + "Selected Reviews/Meta-Analyses"

or something like that... I hope you get the gist

**Review by Kucharssim — CHANGES_REQUESTED** [review]
- [x] https://github.com/jasp-stats/jasp-test-release/issues/1723
- [x] https://github.com/jasp-stats/jasp-test-release/issues/1709
- [ ] https://github.com/jasp-stats/jasp-test-release/issues/1744

So now you display the hierarchy by unenabling the components. I still think that there could be indentation/grouping to communicate it more clearly. Something like this:

```
Distribution plot
  [ ] effect sizes
  [ ] sample sizes
  Options
    [ ] Add density
    [ ] Add rug marks
     ...
```

This is relatively minor, so I let you to decide (as it is admittedly somewhat dependent on personal preference), but I would be glad if you give it a thought.

- [ ] addresses parts of: jasp-stats/jasp-test-release#1698

In general, I like the changes you made. I have one suggestion
Screenshot 2022-01-28 at 15 21 47

It seems to me that the dropdown "Analyze as" does not really fit into that part of the form, which is focused on selecting the studies into the analysis... This option then becomes drowned in the other components and does not really logically relate to the others that surround it.

Maybe move it below the  "Selected Reviews/Meta-Analyses" list, just above the "Analyze data (individually/pooled)"?

**Comment by FBartos** [comment]
ok, I changed all parts according to the suggestions

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Just one small thing

File: R/cochranecommon.R [hunk 3]
```diff
@@ -508,6 +508,12 @@ CochraneCommon   <- function(jaspResults, dataset, options, type) {
     else
       selectedTitles <- sapply(indexing, function(inx) inx[["titleMetaAnalysis"]])[textSearchPositive & !textSearchNegative]
 
+    if (length(selectedTitles) == 0){
+      jaspResults[["selectedOverviewTable"]]$addFootnote(gettext(
+        "There is no matching meta-analysis title for the specified text search", symbol = "Note: "))
```
Comment (Kucharssim) [inline R/cochranecommon.R:513]:
```suggestion
      jaspResults[["selectedOverviewTable"]]$addFootnote(gettext(
        "There is no matching meta-analysis title for the specified text search"), symbol = "Note: ")
```

I believe this is what you wanted...

Comment (FBartos) [reply R/cochranecommon.R:513]: oh thanks, good catch!

**Review by Kucharssim — APPROVED** [review]

**Pull request merged**